code/datums/supplypacks.dm

```diff
@@ -396,7 +396,7 @@ WEAPONS
 					/obj/item/explosive/plastique,
 					/obj/item/explosive/plastique
 					)
-	cost = RO_PRICE_PRICY
+	cost = RO_PRICE_VERY_CHEAP
```

Comment:
`RO_PRICE_NORMAL`

Reply:
...and substantially over-estimate the value of plastique.